### PR TITLE
Make the conditional rule title optional

### PR DESCRIPTION
**Pull request by syldlb**
Until now, each conditions had to have a name property to make a valid form.
We want make this field optional so django-formidable users can decide if they want to use it or not.

## Review

* [x] Tests
* [x] Docs/comments
  * [x] *IN CASE YOU'VE CHANGED THE `formidable.yml` file*: run ``tox -e swagger-statics`` to rebuild the swagger static files **and** commit the diff.
* [x] `CHANGELOG.rst` Updated
* [ ] Delete your branch

### CODE REVIEW

File: formidable/serializers/forms.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ class ConditionTestSerializer(serializers.Serializer):
 
 
 class ConditionSerializer(serializers.Serializer):
-    name = serializers.CharField(required=True)
+    name = serializers.CharField(required=False)
     fields_ids = serializers.ListField(
```
Comment (alexdashkov) [inline formidable/serializers/forms.py]: is this argument required?

File: formidable/forms/conditions.py [hunk 2]
```diff
@@ -51,6 +51,8 @@ def convert_values(field_id, values):
                                    convert_values(test['field_id'],
                                                   test['values']))
                      for test in condition['tests']]
+            if 'name' not in condition:
+                condition['name'] = ''
```
Comment (alexdashkov) [inline formidable/forms/conditions.py]: I'd prefer to use None instead. As it's not empty but doesn't exist.

File: demo/tests/test_conditions.py [hunk 3]
```diff
@@ -87,6 +87,30 @@ def test_jedi_displayed(self):
         self.assertFalse(form.is_valid())
         self.assertIn('bar', form.errors)
 
+    def test_jedi_displayed_without_condition_name(self):
+        self.formidable.conditions = [
+            {
```
Comment (alexdashkov) [inline demo/tests/test_conditions.py]: What's going on if you add two or more rules without names, is it ok?

Comment (syldlb) [reply demo/tests/test_conditions.py]: Yep, I'm going to write some additional tests for better coverage

File: formidable/serializers/forms.py (hunk 1 above)
Comment (syldlb) [reply formidable/serializers/forms.py]: It is not necessary, but "required=False" was used in another serializer, so I thought I could use it here to make it explicit

**Comment by syldlb** [comment]
Github won't let me review the PR because I'm the author, but the settings part looks good to me :+1:

**Comment by alexdashkov** [comment]
@syldlb are you sure that this is a right PR for your comment?

**Comment by brunobord** [comment]
That's ok. please solve conflicts and we'll make a proper release (1.5.0)

**Pull request merged**